File: perl/fizzbuzz.pl
#!/usr/bin/perl 

use strict; 
use warnings;

for my $i (0..100) {
    my $output = "";
    $output .= ($i % 3 == 0) ? "Fizz" : "";
    $output .= ($i % 5 == 0) ? "Buzz" : "";
    $output = ($output eq "") ? $i : $output;
    print("$output\n"); 
}

File: php/fizzbuzz.php
<?php

for ($i = 0; $i <= 100; $i++) { 
    $output = '';
    $output .= ($i % 3 === 0) ? 'Fizz' : '' ;
    $output .= ($i % 5 === 0) ? 'Buzz' : '' ;
    $output = (empty($output)) ? "$i" : $output ;
    echo "$output\n";
}

?>
